spec/models/user_spec.rb

```diff
@@ -151,30 +206,31 @@
       webauth_privgroup_str = 'sdr:administrator-role|sdr:manager-role|sdr:viewer-role'
       mock_webauth = double('webauth', :login => 'asdf', :logged_in? => true, :privgroup => webauth_privgroup_str)
       user = User.find_or_create_by_webauth(mock_webauth)
-      expect(user.is_admin  ).to be_truthy
-      expect(user.is_manager).to be_truthy
-      expect(user.is_viewer ).to be_truthy
-      user.set_groups_to_impersonate(['workgroup:sdr:not-an-administrator-role', 'workgroup:sdr:not-a-manager-role', 'workgroup:sdr:not-a-viewer-role'])
-      expect(user.is_admin  ).to be_falsey
-      expect(user.is_manager).to be_falsey
-      expect(user.is_viewer ).to be_falsey
+      expect(user.is_admin).to be true
```

Comment:
yeah, also agreed w/ @darrenleeweber's reasoning here.  seems fine to prefer truthiness in general, but it also seems reasonable to expect actual boolean values in this specific test.

"This test was previously testing before/after impersonation" would be worrisome... but i don't see it?  i might be missing something, but it still appears to be testing before/after impersonation, as far as i can tell...